x-pack/plugins/timelines/server/search_strategy/index_fields/index.ts

```diff
@@ -158,8 +159,12 @@ export const requestIndexFieldSearch = async (
                 pattern: index,
               });
             }
+            const fieldCapsOptions = request.includeUnmapped
```

Comment:
Overall looks good.

Do you think this warrants adding tests in `x-pack/plugins/timelines/server/search_strategy/index_fields/index.test.ts`

Reply:
Good point, I will add tests to cover new changes!